src/hook/pauseResume.js

```diff
@@ -0,0 +1,299 @@
+/**
+* Copyright (c) 2015 NAVER Corp.
+* egjs projects are licensed under the MIT license
+*/
+eg.module("pauseResume", ["jQuery"], function($) {
+	"use strict";
+
+	var animateFn = $.fn.animate;
+	var stopFn = $.fn.stop;
+	var uuid = 1;
+
+	function now() {
+		return new Date().getTime();
+	}
+
+	function AniProperty(prop, optall, prevProp, isFirst) {
+		this.opt = optall;
+		this.start = -1;
+		this.elapsed = 0;
+		this.paused = false;
+		this.uuid = uuid++;
+		this.easingNames = [];
+		this.prop = isFirst ? {} : $.extend({}, prevProp);
+
+		for (var propName in prop) {
+			var propValue = prop[propName];
+			var markIndex;
+			var sign;
+
+			//If it has a absoulte value.
+			if (typeof(propValue) !== "string" ||
+				(markIndex = propValue.search(/[+|-]=/)) < 0) {
+				this.prop[propName] = propValue;
+				continue;
+			}
+
+			sign = propValue.charAt(markIndex) === "-" ? -1 : 1;
+
+			//If it has a relative value
+			this.prop[propName] = propValue
+				.replace(/([-|+])*([\d|\.])+/g,
+					generateAbsoluteValMaker(prevProp, propName, sign))
+				.replace(/[-|+]+=/g, "");
+		}
+	}
+
+	/**
+	 * Generate a new absolute value maker.
+	 *
+	 * function to avoid JS Hint error "Don't make functions within a loop"
+	 */
+	function generateAbsoluteValMaker(prevProp, propName, sign) {
+		return function absoluteValMaker(match) {
+			var currValue = prevProp[propName] ? parseFloat(prevProp[propName]) : 0;
+			return currValue + Number(match) * sign;
```

Comment:
When It multiply string by number result is number. So IMO Number casting is unnecessary.